### ISSUE TITLE
Created files/folders no longer appear in tree view automatically

### ISSUE THREAD

**Issue by patrickhulce** [dkundel/vscode-new-file#54]
Wanted to give you a heads up that in the latest VSCode Insiders build, files/folders created via the new file command here no longer show up automatically in the File Explorer/Tree View. I now have to explicitly refresh in order to see the created files/folders.

Not sure if it's on their side or yours, but it's been two consecutive builds now, so thought you should know. Great work my friend!

#### Build Info
```
Version 1.31.0-insider (1.31.0-insider)
9fe48f15b1a252a0e9e1ef5975301203baf10b8a
2019-01-25T08:04:51.283Z
```

**Comment by jasonsbarr** [comment]
Having the same problem. I just installed this extension because patbenatar/advanced-new-file was giving me errors I couldn't fix, so at least this one is working at all. Still annoying, though.

**Comment by jasonsbarr** [comment]
The plot thickens: I erased all `newFile` settings from `settings.json` and now the new files are showing up in the tree.

Edit: after removing all settings and then adding them back it continues to show new files in the tree.

**Comment by jasonsbarr** [comment]
Ok, I figured out why the files weren't showing up: somehow, whether it's because I didn't understand how the extension works or because something's buggy, I ended up creating the files in my home directory instead of the project directory. Are you 100% sure you're creating the files in a directory that's in your tree?

**Comment by patrickhulce** [comment]
> Are you 100% sure you're creating the files in a directory that's in your tree?

Yep when I refresh the file tree they start to show up. Uninstalling and reinstalling the extension along with removing the settings.json actually fixed it for me too 🎉 thanks for the tip!

**Comment by jasonsbarr** [comment]
Great, I'm glad my shot in the dark was useful! 😃
